_blog/why-null-models-crispr.md

```diff
@@ -0,0 +1,84 @@
+
+[^3]: This [theory paper](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005486) included variable spacer effectiveness as an ingredient to get broad abundance distributions.
+
+<p align="center">
+<img src="/img/posts/why-null-models-crispr/Heler2019.svg" alt="Heler2019" width="450px" style="padding:5px;">
+</p>
+
+So we're done, right? We know why some spacers are highly abundant and others are not, and if we see a broad abundance distribution we know the cause. Or do we? We've answered the question "does process A cause result B", but what about the other direction? Remember where we started: to figure out what's going on when we observe something, we have to rule out other explanations. Could there be other explanations for a spacer abundance distribution that looks like this that would make it hard to say for sure that B is evidence of A?
+
+Enter our null model. We modelled bacteria and phage interacting with CRISPR, and we included absolutely no distinguishing features for individual spacers: all spacers were equally likely to be acquired and all spacers were equally effective at preventing infection. This is the most boring model possible that still includes the bare bones of CRISPR immunity. We simulated this population over time, watching bacteria grow, die, and acquire spacer sequences randomly. If a bacterium had a spacer when it was infected, it got a CRISPR immune boost and was more likely to survive.
+
+<p align="center">
+<img src="/img/posts/why-null-models-crispr/model_diagram_simplified.png" alt="model" width="800px" style="padding:5px;">
+</p>
+
+Amazingly, we found a strikingly broad spacer abundance distribution. The figure below compares experimental data on the left and our simulation data on the right. Our null model shows that you don't need special spacer sequences to find that some are very common and some are rare: you can get large differences in abundance purely because of randomness[^4].
```

Comment:
Maybe explain what you mean by broad to non-experts? I like how you describe 'long-tail' earlier... along those lines?